### PR TITLE
SPV_QCOM_image_processing, fixes #307

### LINKED ISSUE #307: Missing capabilities in spirv.core.grammar.json
The Vulkan 1.3.222 spec added a reference to three new SPIR-V capabilities to vk.xml:
- TextureSampleWeightedQCOM
- TextureBoxFilterQCOM
- TextureBlockMatchQCOM

These capabilities are currently missing from the spirv.core.grammar.json file.

### PR DESCRIPTION
**Pull request by wooyoungqcom**
Fix #307

### CODE REVIEW

**Comment by wooyoungqcom** [comment]
Merged SPV_QCOM_image_processing changes to the main

File: include/spirv/unified1/spirv.core.grammar.json [hunk 1]
```diff
@@ -4592,6 +4592,70 @@
...
+      "opcode" : 4480,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Texture'" },
+        { "kind" : "IdRef", "name" : "'Coordinates'" },
+        { "kind" : "IdRef", "name" : "'Weights'" }
+      ],
+      "capabilities" : [ "TextureSampleWeightedQCOM" ],
+      "extensions" : [ "SPV_QCOM_image_processing" ],
```
Comment (alan-baker) [inline include/spirv/unified1/spirv.core.grammar.json:4607]: Please don't include extensions if the capability is enabled by that extensions.

**Review by alan-baker — CHANGES_REQUESTED** [review]

**Comment by wooyoungqcom** [comment]
Updated the pull request with the "capabilities" removed.

The successful merge of the pull request will resolve the issue #307

**Comment by wooyoungqcom** [comment]
Updated the pull request with the "extension" removed as requested.

File: include/spirv/unified1/spirv.core.grammar.json [hunk 2]
```diff
@@ -4592,6 +4592,67 @@
...
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'Target'" },
+        { "kind" : "IdRef", "name" : "'Target Coordinates'" },
+        { "kind" : "IdRef", "name" : "'Reference'" },
+        { "kind" : "IdRef", "name" : "'Reference Coordinates'" },
+        { "kind" : "IdRef", "name" : "'Block Size'" }
+      ],
+      "capabilities" : [ "TextureBlockMatchQCOM" ],
+      "extensions" : [ "SPV_QCOM_image_processing" ],
```
Comment (alan-baker) [inline include/spirv/unified1/spirv.core.grammar.json:4653]: Extension is still present here and should be removed.

**Review by alan-baker — CHANGES_REQUESTED** [review]

File: include/spirv/unified1/spirv.core.grammar.json (hunk 2 above)
Comment (wooyoungqcom) [reply include/spirv/unified1/spirv.core.grammar.json]: Thanks, Alan. I missed one. It is updated.

**Review by alan-baker — APPROVED** [review]

**Pull request merged**